.travis.yml

```diff
@@ -142,25 +178,34 @@ matrix:
         # Following entries used to be included in testing, but they only proved useful while changing things in CMake setup.
         # They are meant to be used when there are changes to CMakeLists.txt
         # - os: linux
-        #   env: OPENRCT2_CMAKE_OPTS="-DDISABLE_NETWORK=ON -DDISABLE_HTTP_TWITCH=ON -DCMAKE_C_COMPILER=gcc-4.8 -DCMAKE_CXX_COMPILER=g++-4.8"
+        #   env: OPENRCT2_CMAKE_OPTS="-DENABLE_NETWORK=NO -DENABLE_HTTP_TWITCH=NO -DCMAKE_C_COMPILER=gcc-4.8 -DCMAKE_CXX_COMPILER=g++-4.8"
```

Comment:
These were not tested in a looooong time… They should be tested at least once now or removed. GCC 4.8 is probably not going to fulfil our C++14 needs now.